app/[contentType]/page.tsx

```diff
@@ -1,7 +1,7 @@
 import { Metadata } from 'next';
 import { notFound } from 'next/navigation';
 import { Container, ContentList } from '../../components';
-import { site } from '../../config/index.json';
+import info from '../../config/index.json' assert { type: 'json' };
```

Comment:
Named imports throw this Webpack error, and to solve this issue, we have to switch to `default imports` 
```
./app/page.tsx
Should not import the named export 'site'.'siteTitle' (imported as 'site') from default-exporting module (only default export is available soon)
```

> If a module import has an attribute with key type and value json, the host is required to either fail the import, or treat it as a JSON module. Specifically this means that the content of the module is parsed as JSON and the resulting JSON object is the default export of the module (which has no named exports).

https://github.com/tc39/proposal-json-modules?tab=readme-ov-file#proposed-semantics-and-interoperability